test/unit/proxy/cpsubsystem/FencedLockProxyTest.js

```diff
@@ -62,7 +66,7 @@ describe('FencedLockProxyTest', function () {
 
     function stubRequest(methodName, result, firstCallErr) {
         let called = 0;
-        const stub = sandbox.stub(proxy, methodName).callsFake(() => {
+        return sandbox.stub(proxy, methodName).callsFake(() => {
```

Comment:
these are obviously useless variables that are safe to delete and return directly